src/components/BookForm.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const BookForm = () => {
+  const title = 'ADD NEW BOOK';
+  return (
+    <div>
+      <section>
+        <h2>{title}</h2>
+        <form>
+          <input type="text" placeholder="Add title" className="input-text" name="title" />
+          <input type="text" placeholder="Add author" className="input-text" name="author" />
+          <button type="submit">ADD BOOK</button>
+        </form>
+      </section>
+    </div>
```

Comment:
- [ ] **[OPTIONAL]:** I would suggest that you don't use two wrapper components, in this case, using just the `<section>` would be the better semantic option, because you're describing the "Add new book" section! 🤓 🔧 

```suggestion
    
      <section>
        <h2>{title}</h2>
        <form>
          <input type="text" placeholder="Add title" className="input-text" name="title" />
          <input type="text" placeholder="Add author" className="input-text" name="author" />
          <button type="submit">ADD BOOK</button>
        </form>
      </section>
    
```